app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    User.take
```

Comment:
- [ ] The application record has a relation to the migration, I don't know what the **user.take** will do but it certainly does not render the user in your database.